src/components/TextInput.vue

```diff
@@ -101,6 +101,9 @@ export default class TextInput extends Vue {
   @Prop({ type: Boolean })
   readonly!: boolean;
 
+  @Prop({ type: String, default: "off" })
```

Comment:
Should this be a Boolean?

Reply:
Doesn't look like it, it needs to be string wise 'off' or 'on', found that strange myself o.o
Using a boolean true / false doesn't work.